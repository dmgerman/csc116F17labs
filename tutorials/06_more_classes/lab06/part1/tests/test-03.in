 3 5
 1 0 0 2 3
 0 1 0 4 5
 0 0 1 6 7