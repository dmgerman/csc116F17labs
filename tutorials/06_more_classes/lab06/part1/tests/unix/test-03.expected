Reading matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 

Successfully read a matrix with dimensions 3 x 5
M: 
1 0 0 2 3 
0 1 0 4 5 
0 0 1 6 7 
M is not diagonal.
Unable to compute trace: Unable to compute trace of a non-square matrix
